api/user/utils.ts

```diff
@@ -29,11 +29,10 @@ export function isCurrentUserCaseAllocator(currentUserRoleAssignment: RoleAssign
                                            currentJurisdiction?: string,
                                            currentLocation?: string): boolean {
   return !!currentUserRoleAssignment
-   && currentUserRoleAssignment.roleType === ORGANISATION_ROLE_TYPE
-   && currentUserRoleAssignment.roleName === CASE_ALLOCATOR_ROLE
-   && (!currentJurisdiction || currentUserRoleAssignment.attributes.jurisdiction === currentJurisdiction)
-   // primaryLocation changed to baseLocation here
-   && (!currentLocation || currentUserRoleAssignment.attributes.baseLocation === currentLocation);
+    && currentUserRoleAssignment.roleType === ORGANISATION_ROLE_TYPE
+    && currentUserRoleAssignment.roleName === CASE_ALLOCATOR_ROLE
+    && (!currentJurisdiction || currentUserRoleAssignment.attributes.jurisdiction === currentJurisdiction)
+    && (!currentLocation || currentUserRoleAssignment.attributes.baseLocation === currentLocation);
 }
```

Comment:
it seems in this method nothing has been changed apart from removing a comment, I would try and refrain from doing this to simplify the code reviews.